### DISCUSSION TITLE
Github Student Pack Help

### DISCUSSION THREAD

**Discussion by Bilal-Nadeem** [community/community#47233] [GitHub Education] [answered]
I am a university student trying to get GitHub student pack but I have the following errors. 
- First of all it says the university doesn't exist in my location even though I'm applying right from it. (This is the main problem) 
- Secondly my student ID has things like date on the back and rest of it on the front so one side won't have all the information in picture. 
- Third is that I can only use front facing camera to take pictures so it's hard to take pick of the ID and it also results in low quality image. 

Thank you!

**Comment by caitlon** [comment, marked as answer]
Hi, I've got a student packet myself just few hours ago and I also had problems filling out the data. So, what I've done:
1. I found out that GitHub mistakenly detected my location when I had a VPN enabled (for the same country where I am) - then I turned it off, after GitHub considered my location to be correct.
2. Instead of taking a picture with the camera of the proof of education or the ID student card - I sent a confirmation of education in png format: I downloaded the automatically generated confirmation from the university website, took a screenshot, converted it to the right format, and there were no problems. (the confirmation of education was not in English)

**Reply by samus-aran** [reply]
Appreciate you replying and helping out @caitlon 👍 
(edited!)

**Comment by danielcregg** [comment]
I am a lecturer. I have a teacher account. Can I get access to the benefits of a student account such as a free domain name?

**Comment by SKSHOHAN1111** [comment]
Hallo

**Comment by abdullahh01** [comment]
Hello I am an student i need an datacamp subscription as i want to do more courses please i have 2 to 3 courses of data science and i want to do more

**Comment by Nanda-Kiran** [comment]
HI, I TRIED TO APPLY USING THE COLLAGE  BONAFIDE CERTIFICATE AS MY ID DOESN'T HAVE DATE. BUT IT SHOWING THAT  IT NEEDS COLLED ID.
 AND MY LOCATION IS JUST 500M FROM COLLEGE BUT IT IS SHOWING ERROR.
 AND HOW TO UPLOAD PHOTO INSTEAD OF TAKING?

**Comment by trdunsworth** [comment]
I'm running into the same problem. I reached out to the university and received a letter stating I'm currently enrolled and in good standing and that they anticipate my graduation date to be December 2024. I understand not accepting my student ID because it does not have a date on it. I'm in the situation where my university was absorbed by another so the letterhead is for the new university. According to the AI-evaluator, I live too far from my campus. However, there are no classes on a campus and the university's template will not indicate students are not local because no students are local. 

Can you folks to something/anything to address this?
